templates/shared/forms/interactive/aws.html

```diff
@@ -3,7 +3,7 @@
     <header class="p-modal__header" style="display: block; border-bottom: 0;">
       <button class="p-modal__close" aria-label="Close active modal" style="margin-left: -1rem">Close</button>
       <h2 class="p-modal__title u-sv1" id="modal-title">Canonical - the "Ubuntu on AWS" experts</h2>
-      <p>Already running Ubuntu on AWS, or planning a migration to the public cloud? Canonical has expertise in the right areas to assist you in your project, whether it involves application architecture, optimization or streamlining operations at scale. Share with us some detail on your project and let's get started.</p>
+      <p>Already running Ubuntu on AWS, or planning a migration to the public cloud? Canonical has the expertise to assist you in your project, whether it involves compliance, application architecture, optimisation or streamlining operations at scale. Share some details about your project with us and let's get started.</p>
```

Comment:
That will be because `smartquotes` runs on page load and so only affects static content. This content is loaded and injected in the page via JS. Therefore, it's better to encode these ones. We could switch the system to include the content in the template from the server.

Reply:
@albertkol this content should be below the header. So move it down one line.

Reply:
That makes sense thanks

Reply:
Done.